docker/frontend/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM node:lts-alpine3.10
```

Comment:
Don't use outdated alpine

Reply:
If you're making custom image you need a way to push/pull it

Reply:
Lts version just changed and all builds are broken

Reply:
I prefer to move with https://github.com/skilld-labs/docker-frontend/pull/26 cos themes are all separate and they must manage its dependencies out of sdc

Reply:
agreed, it's a dependency that is dependent solely on our theme(s) after fall